.travis.yml

```diff
@@ -21,7 +21,7 @@ before_install:
   - mysql < build/database/test_db_setup.sql
 
 install:
-  - python setup.py install
+  - pip install -e .
```

Comment:
Could it be useful in include a short comment on who pip install -e . does as different from pip install -r requirements.txt?